runtime/ts/storage/firebase-storage.ts

```diff
@@ -346,7 +346,9 @@ class FirebaseVariable extends FirebaseStorageProvider {
     this.version = data.version;
 
     this.resolveInitialized();
-    this._fire('change', {data: data.value, version: this.version});
+    // Firebase doesn't maintain a distinction between null and undefined, but we explicitly
+    // require empty variables to store 'null'.
+    this._fire('change', {data: data.value || null, version: this.version});
```

Comment:
If `data.value` is `false`, as written this will set `null` into the `data` field of the fired object. So we'd lose the primitive `Boolean` value. Maybe we won't get `false` Booleans out of Firebase, or `null` would still be ok as it's falsy, but I figured worth checking.

Reply:
data.value can either be a tree location or undefined. We expect it to have id and rawData if it's the former, so false would already be an error.